coalib/parsing/ConfParser.py

```diff
@@ -156,7 +157,12 @@ def __parse_lines(self, lines, origin):
                                     remove_empty_iter_elements=
                                     self.__remove_empty_iter_elements),
                                     # Stop ignoring
-                            allow_appending=(keys == []))
+                            allow_appending=(
+                                keys == [] or keys == [('', 'ignore')]))
```

Comment:
you use `[('', 'ignore')]` again here.

another reason why it should be a constant.